couchbase/README.md

```diff
@@ -32,12 +32,12 @@ The Couchbase check is included in the [Datadog Agent][3] package, so you don't
 1.  Add this configuration block to your `couchbase.d/conf.yaml` file to start gathering your [Couchbase Metrics](#metrics):
 
     ```
-    init_config:
+      init_config:
```

Comment:
Why are you adding indent here?
Also can @DataDog/baklava confirm the impact of this for our docs ?